service/confirmation-repeater/src/helpers.ts

```diff
@@ -1,14 +1,16 @@
-const dotenv = require('dotenv');
-dotenv.config();
-
-let retryArray: string[] = [];
-if (process.env.RETRY_INTERVAL) retryArray = process.env.RETRY_INTERVAL.split(",");
+let retryArray: number[] = [];
+if (process.env.RETRY_INTERVAL) retryArray = process.env.RETRY_INTERVAL.split(",").map(x => parseInt(x)).filter(x => x > 0);
```

Comment:
@1v4n4 pls parse input (to number) on reading it, and later have already parsed input in variables. otherwise you will have potentially unknown data in the bloodstream